setup.py

```diff
@@ -0,0 +1,75 @@
+from setuptools import setup, find_packages
+
+__version__ = "0.1.0"
+
+# More information on setting values:
+# https://github.com/Sceptre/project/wiki/sceptre-template-handler-template
+
+# lowercase, use `-` as separator.
+TEMPLATE_HANDLER_NAME = 'custom-template-handler'
+# the template_handler call in sceptre e.g. !command_name.
+TEMPLATE_HANDLER_COMMAND_NAME = 'custom_template_handler'
+# do not change. Rename template_handler/template_handler.py to template_handler/{TEMPLATE_HANDLER_COMMAND_NAME}.py
+TEMPLATE_HANDLER_MODULE_NAME = 'template_handler.{}'.format(TEMPLATE_HANDLER_COMMAND_NAME)
+# CamelCase name of template_handler class in template_handler.template_handler.
+TEMPLATE_HANDLER_CLASS = 'CustomTemplateHandler'
+# One line summary description
+TEMPLATE_HANDLER_DESCRIPTION = ''
+# if multiple use a single string with comma separated names.
+TEMPLATE_HANDLER_AUTHOR = 'Sceptre'
+# if multiple use single string with commas.
+TEMPLATE_HANDLER_AUTHOR_EMAIL = 'sceptre@sceptre.org'
+TEMPLATE_HANDLER_URL = 'https://github.com/sceptre/{}'.format(TEMPLATE_HANDLER_NAME)
+
+with open("README.md") as readme_file:
+    README = readme_file.read()
+
+install_requirements = [
+    "packaging==16.8",
+    "requests>=2.25,<3"
```

Comment:
These seem odd to me. Why are these required on every template handler project?

Reply:
removed.